libs/vre/shared/app-resource-page/src/lib/resource-representation.component.ts

```diff
@@ -15,25 +15,23 @@ import { filter, map } from 'rxjs/operators';
 
 @Component({
   selector: 'app-resource-representation',
-  template: ` <div class="representation-container center" [ngSwitch]="representationToDisplay.fileValue.type">
-    <app-still-image
-      #stillImage
-      class="dsp-representation stillimage"
-      [attachedProject]="attachedProject$ | async"
-      *ngSwitchCase="representationConstants.stillImage"
-      [resource]="resource.res" />
-    <app-still-image
-      #stillImage
-      class="dsp-representation stillimage"
-      *ngSwitchCase="representationConstants.externalStillImage"
-      [attachedProject]="attachedProject$ | async"
-      [resource]="resource.res" />
-
+  template: ` <div class="representation-container center" [ngSwitch]="true">
+    <ng-container *ngIf="attachedProject$ | async as attachedProject">
+      <app-still-image
+        #stillImage
+        class="dsp-representation stillimage"
+        *ngSwitchCase="
+          representationToDisplay.fileValue.type === representationConstants.stillImage ||
+          representationToDisplay.fileValue.type === representationConstants.externalStillImage
+        "
+        [attachedProject]="attachedProject"
+        [resource]="resource.res" />
+    </ng-container>
```

Comment:
```suggestion
  template: ` <div class="representation-container center" [ngSwitch]="representationToDisplay.fileValue.type">
    <ng-container *ngIf="attachedProject$ | async as attachedProject">
      <app-still-image
        #stillImage
        class="dsp-representation stillimage"
        *ngSwitchCase="representationConstants.stillImage || representationConstants.externalStillImage"
        [attachedProject]="attachedProject"
        [resource]="resource.res" />
    </ng-container>
```

Reply:
@mpro7 I mentioned that angular templates are stupid, it doesn't understand this